repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -52,7 +52,16 @@ object ProjectUpdateRequest {
   implicit val updateRequestFormat: OFormat[ProjectUpdateRequest] = Json.format[ProjectUpdateRequest]
 }
 
-final case class Version(value: String) extends AnyVal
+final case class Version(name: String, message: String, target: String, commit: Commit)
+object Version {
+  implicit val versionPlayFormat: OFormat[Version] = Json.format[Version]
+  implicit val versionsPlayListFormat: Reads[Seq[Version]] = Reads.seq(versionPlayFormat)
```

Comment:
No need for a reader of an object sequence.
If you do have a formatter for a single object, `play-json` must be able to handle other things automatically.

Reply:
Very strange but everything works now. Fixed.